server/src/db/migrations/20230502144326_createDogParks.cjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("dogParks", (table) => {
+        table.bigIncrements("id")
+        table.string("name").notNullable()
+        table.text("description").notNullable()
+        table.string("address").notNullable()
+        table.string("neighborhood")
+        table.boolean("hasTrash")
+        table.boolean("hasBags")
+        table.boolean("hasFence")
+        table.boolean("hasWater")
+        table.boolean("hasBenches")
```

Comment:
its good practice to always have a `defaultTo` and `notNullable` for all boolean fields. That way, `null` can never be added as a value to these columns (which would lead to false negatives) 
Update these fields to have both, and discuss with your group whether each of these booleans should default to true or false